docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Beckn/Beckn-enabled-Reputation-Infrasructure
+---
+
+
+# Author
+Monishka Jha
+
+# Contact Info
+[Email](jhamonishka@gmail.com) 
```

Comment:
add `mailto:` before your email

Reply:
@photon0205 Did thechange Can you check the PR now